src/frame.js

```diff
@@ -0,0 +1,27 @@
+class Frame {
+  strike (roll) {
+    if (roll === 10) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  spare (roll1, roll2) {
+    if (roll1 + roll2 === 10) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  calcBonus (roll1, roll2) {
+    if (this.strike(roll1) === true) {
+      return 2;
+    } else if (this.spare(roll1, roll2)) {
+      return 1;
+    } else {
+      return 0;
+    }
+  }
+}
```

Comment:
Frame class is very clean, with no duplication.

One thing you could look at is reducing the strike and spare classes to one line. 

I think I understand how your bonus mechanism works, and I like how you are assigning a number so it knows how much bonus needs to be applied'